easybuild/easyconfigs/h/HDF5/HDF5-1.8.18-foss-2016b-serial.eb

```diff
@@ -0,0 +1,26 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+
+name = 'HDF5'
+version = '1.8.18'
+versionsuffix = '-serial'
+
+homepage = 'https://support.hdfgroup.org/HDF5/'
+description = """HDF5 is a unique technology suite that makes possible the management of
+ extremely large and complex data collections."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True, 'usempi': False}
+
+source_urls = ['https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-%(version_major_minor)s/hdf5-%(version)s/src']
+sources = [SOURCELOWER_TAR_GZ]
+
+checksums = ['cdb195ad8d9e6782acf24b2488061289f615628c2ccda8457b0a0c3fb7a8a063']
+
+buildopts = 'CXXFLAGS="$CXXFLAGS -DMPICH_IGNORE_CXX_SEEK"'
```

Comment:
this is taken care of by the HDF5 easyblock now, see https://github.com/hpcugent/easybuild-easyblocks/pull/1200

also, it's not needed at all with `foss` as toolchain

so, please remove